contracts/facets/EmbalmerFacet.sol

```diff
@@ -311,6 +356,126 @@ contract EmbalmerFacet {
+        uint256 storageFees = LibBonds.calculateTotalFees(
+            identifier,
+            archaeologistAddresses
+        );
+
+        // Transfer the total fees back to the embalmer
+        s.sarcoToken.transfer(
+            s.sarcophaguses[identifier].embalmer,
+            storageFees
+        );
+
+        // Emit an event
+        emit LibEvents.CancelSarcophagus(identifier);
+
+        return true;
+    }
+
+    /// @notice Permanently closes the sarcophagus, giving it no opportunity to
+    /// be resurrected.
+    /// This may only be done after finalizeSarcophagus and before the
+    /// resurrection time has passed.
+    /// @dev Extends the resurrection time into infinity so that that unwrap
+    /// will never be successful.
+    function burySarcophagus(bytes32 identifier) external returns (bool) {
+        // Confirm that the sender is the embalmer
+        if (s.sarcophaguses[identifier].embalmer != msg.sender) {
+            revert LibErrors.SenderNotEmbalmer(
+                msg.sender,
+                s.sarcophaguses[identifier].embalmer
+            );
+        }
+
+        // Confirm that the sarcophagus exists
+        if (
+            s.sarcophaguses[identifier].state !=
+            LibTypes.SarcophagusState.Exists
+        ) {
+            revert LibErrors.SarcophagusDoesNotExist(identifier);
+        }
+
+        // Confirm that the sarcophagus is finalized by checking if there is an
+        // arweaveTxId
+        if (bytes(s.sarcophaguses[identifier].arweaveTxId).length == 0) {
+            revert LibErrors.SarcophagusNotFinalized(identifier);
+        }
+
+        // Confirm that the current resurrection time is in the future
+        if (s.sarcophaguses[identifier].resurrectionTime <= block.timestamp) {
+            revert LibErrors.ResurrectionTimeInPast(
+                s.sarcophaguses[identifier].resurrectionTime
+            );
+        }
+
+        // Set resurrection time to infinity
+        s.sarcophaguses[identifier].resurrectionTime = 2**256 - 1;
+
+        // Set sarcophagus state to done
+        s.sarcophaguses[identifier].state = LibTypes.SarcophagusState.Done;
+
+        // For each archaeologist on the sarcophagus,
+        // 1. Unlock their cursed bond
+        // 2. Transfer digging fees to the archaeologist.
+        address[] memory archaeologistAddresses = s
+            .sarcophaguses[identifier]
+            .archaeologists;
+
+        for (uint256 i = 0; i < archaeologistAddresses.length; i++) {
+            // Unlock the archaeologist's cursed bond
+            LibBonds.freeArchaeologist(identifier, archaeologistAddresses[i]);
+
+            // Transfer the digging fees to the archaeologist
+            s.sarcoToken.transfer(
+                archaeologistAddresses[i],
+                getArchaeologist(identifier, archaeologistAddresses[i])
+                    .diggingFee
+            );
+        }
```

Comment:
I realize this is the same functionality as in `v1` -- but what happens to the bounty the embalmer locked up for the sarcophagus? Is that in the contract forever? 

Wondering if we should either transfer that back to the embalmer or potentially make it a protocol fee.